src/Nri/Ui/SegmentedControl/V6.elm

```diff
@@ -0,0 +1,144 @@
+module Nri.Ui.SegmentedControl.V6 exposing (Config, Icon, Option, Width(..), view)
+
+{-|
+
+@docs Config, Icon, Option, Width, view
+
+-}
+
+import Accessibility.Styled exposing (..)
+import Accessibility.Styled.Role as Role
+import Css exposing (..)
+import Css.Foreign exposing (Snippet, adjacentSiblings, children, class, descendants, each, everything, media, selector, withClass)
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes as Attr exposing (css)
+import Html.Styled.Events as Events
+import Nri.Ui.Colors.Extra exposing (withAlpha)
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.CssFlexBoxWithVendorPrefix as FlexBox
+import Nri.Ui.Fonts.V1 as Fonts
+import Nri.Ui.Icon.V3 as Icon
+
+
+{-| -}
+type alias Config a msg =
+    { onClick : a -> msg
+    , options : List (Option a)
+    , selected : a
+    , width : Width
+    }
+
+
+{-| -}
+type alias Option a =
+    { value : a
+    , icon : Maybe Icon
+    , label : String
+    , id : String
+    }
+
+
+{-| -}
+type Width
+    = FitContent
+    | FillContainer
+
+
+{-| -}
+type alias Icon =
+    { alt : String
+    , icon : Icon.IconType
+    }
+
+
+{-| -}
+view : Config a msg -> Html.Html msg
+view config =
+    config.options
+        |> List.map
+            (\option ->
+                Html.div
```

Comment:
🎨 Could we use an `Nri.Ui.styled div` here that points to "segmented-control-v6-tab" or something? Looks like we are pretty well covered with accessibility attributes, but I still find the `data` attrs helpful